lib/silencer/rails/environment.rb

```diff
@@ -8,7 +8,7 @@ module Environment
     module_function
 
     def rails?
-      defined?(::ActionDispatch)
+      defined?(::ActionDispatch) || defined?(::Rails)
```

Comment:
I think this could be reverted to simply `defined?(::Rails)`, since I don't think there'd be a situation in which `::ActionDispatch` would be undefined but `::Rails` would be defined.

Reply:
Good point. I wasn't entirely sure which one to check for. Either probably works and two is probably redundant.